src/herder/PendingEnvelopes.cpp

```diff
@@ -563,18 +563,13 @@ PendingEnvelopes::slotClosed(uint64 slotIndex)
 TxSetFramePtr
 PendingEnvelopes::getTxSet(Hash const& hash)
 {
-    if (mTxSetCache.exists(hash))
-    {
-        return mTxSetCache.get(hash).second;
-    }
-
-    return TxSetFramePtr();
+    return getKnownTxSet(hash, 0, true);
 }
 
 SCPQuorumSetPtr
 PendingEnvelopes::getQSet(Hash const& hash)
 {
-    auto qset = getKnownQSet(hash, true);
+    auto qset = getKnownQSet(hash, false);
```

Comment:
Why did you change this to false? This doesn't match the behavior of `getTxSet` and it also doesn't match the old behavior of `getQSet` prior to this PR.